lib/solargraph/rails/delegate.rb

```diff
@@ -0,0 +1,38 @@
+module Solargraph
+  module Rails
+    class Delegate
+      def self.instance
+        @instance ||= self.new
+      end
+
+      def process(source_map, ns)
+        return [] unless source_map.code.include?('delegate')
+
+        walker = Walker.from_source(source_map.source)
+        pins = []
+
+        walker.on :send, [nil, :delegate] do |ast|
+          methods =
+            ast.children[2..-1]
+              .map { |c| c.children.first }
```

Comment:
Is it possible to use `#filter_map` in this case instead of `#select` and `#map`? `#filter_map` seems to be a bit faster ([source](https://blog.saeloun.com/2019/05/25/ruby-2-7-enumerable-filter-map.html))

Reply:
not worth the effort, we don't have a microbenchmark contest here